openid-4-verifiable-presentations-1_0.md

```diff
@@ -1864,6 +1923,75 @@ Wallets SHOULD NOT fetch URLs provided by the Verifier in a request that are unk
 
+## Use, Retention and Disclosure Limitation {#use_retention_and_disclosure_limitation}
+
+### Unobservability by Wallets {#unobservability_by_wallets}
+
+*   **No Data Collection or Monitoring**: Wallet providers are prohibited from collecting or monitoring data about how users interact with their wallets, digital IDs, or stored documents. This means providers cannot track user behavior or transactions. 
+    
+*   **User-Only Visibility**: Only the user can view their transaction history and data within the wallet. Even issuers of digital documents (e.g., IDs or licenses) are not informed when these documents are used. 
+    
+*   **Prevention of Profiling and Tracking:** The wallet's design incorporates data minimization principles, ensuring that only essential information is shared during interactions. This limits opportunities for profiling or linking user activities across different services
+
+### Verifier's Use, Retention and Disclosure Limitation {#verifiers_use_retntion_and_disclosure_limitation}
+
+Verifier should
+
+*  limit the use, retention and disclosure of PII to that which is necessary in order to fulfil specific, explicit and legitimate purposes communicated to the user; 
+*  retain PII only as long as necessary to fulfil the stated purpose or as long as it is valid; and 
+*  lock any PII when and for as long as the stated purposes have expired, but where applicable laws require retention. 
+    
+## Accuracy and Quality {#accuracy_and_quality}
+
+* Verifiers ensure that the PII processed is accurate, complete, up-to-date (unless there is a legitimate basis for keeping outdated data), adequate and relevant for the purpose of use. This could be achieved by only accepting credentials from trusted issuers and checking the issuance and expiry date.
+* If making changes to the already collected and stored data, the verifier should make sure that the new presentation is about the same user.
+* Verifiers should establish a means to detect status change (e.g. revocation) of the obtained credentials through the OpenID4VP protocol. (e.g., through StatusList.) 
+    
+## Openness, Transparency and Notice {#openness_transparency_and_notice}
+
+*   Wallet should keep track of the verifiers’ privacy notices and make them readily available to the user.   
+*   Wallet should make its privacy notices readily available to the user. 
+*   A privacy notice should at least include:  
+    *   the specified PII required for the specified purpose;
+    *   the specified purpose for PII collection;
+    *   the specified processing (including collection, communication and storage mechanisms);
+    *   the types of authorized natural persons who will access the PII and to whom the PII can be transferred; and
+    *   the specified PII retention and disposal requirements.
+        
```

Comment:
```suggestion
```

Reply:
While DCQL has lost "purpose", there are still some relevant text in 5.11 verifier attestations. 
Perhaps we could rephrase to something like: 

* Verifier Attestations allow verifier to communicate context of the use of PII such as acceptable use, retention periods, or access rights etc. to improve openness and transparency.